crates/providers/nats/Cargo.toml

```diff
@@ -15,11 +15,14 @@ async-trait = { workspace = true }
 base64 = { workspace = true }
 bytes = { workspace = true }
 futures = { workspace = true }
+lazy_static = "1"
```

Comment:
Please use workspace dependencies. (`crates/providers`) contain their own workspace.

This particular dependency should have a TODO about replacing it with `once_cell`

Reply:
This is actually where I am curious. I've never done a Rust monorepo (just one with a few crates). I had understood workspace dependencies to be the ones shared across all crates and the ones in here are specific to this provider. Is the proper procedure to put _all_ of them at the top level?

Reply:
I lifted them all, but still curious about my question